projects/swimlane/ngx-datatable/package.json

```diff
@@ -6,7 +6,7 @@
     "@angular/common": ">=11.0.2",
     "@angular/core": ">=11.0.2",
     "@angular/platform-browser": ">=11.0.2",
-    "rxjs": "^6.6.3"
+    "rxjs": "^6.6.3 || ^7.4.0"
```

Comment:
This is the same versions accepted by the latest Angular 13: https://github.com/angular/angular/blob/3b91101bbca29bb2d6a1765fa1fde340cd6dcc41/packages/common/package.json#L24